Spanner/src/SpannerClient.php

```diff
@@ -252,11 +252,22 @@ public function __construct(array $config = [])
      * $batch = $spanner->batch('instance-id', 'database-id');
      * ```
      *
+     * ```
+     * Database role configured in the optional $options array
+     * will be applied to the session created by this object.
+     * $batch = $spanner->batch('instance-id', 'database-id', ['databaseRole' => 'Reader']);
+     * ```
```

Comment:
```suggestion
     * Database role configured in the optional $options array
     * will be applied to the session created by this object.
     * ```
     * $batch = $spanner->batch('instance-id', 'database-id', ['databaseRole' => 'Reader']);
     * ```
```

Reply:
Done